### ISSUE TITLE
SRW - Error: [$compile:multidir]

### ISSUE THREAD

**Issue by mrakitin** [radiasoft/sirepo#694]
We with @ochubar observed the following error on our server:
```js
{
    "cause": "<div data-ng-view=\"\">",
    "message": "Error: [$compile:multidir] http://errors.angularjs.org/1.6.1/$compile/multidir?p0=beamlineItemEditor&p1=%20(module%3A%20SirepoApp)&p2=beamlineItemEditor&p3=%
20(module%3A%20SirepoApp)&p4=new%2Fisolated%20scope&p5=%3Cdiv%20id%3D%22srw-aperture-editor%22%20data-model-name%3D%22aperture%22%20data-beamline-item-editor%3D%22%22%3E",
    "stackTrace": [
        "$@https://expdev.nsls2.bnl.gov/static/js/ext/angular-1.6.1.min.js?20170214.000000:80:202",
        "X@https://expdev.nsls2.bnl.gov/static/js/ext/angular-1.6.1.min.js?20170214.000000:70:232",
        "Na@https://expdev.nsls2.bnl.gov/static/js/ext/angular-1.6.1.min.js?20170214.000000:61:165",
        "Na@https://expdev.nsls2.bnl.gov/static/js/ext/angular-1.6.1.min.js?20170214.000000:61:253",
        "Na@https://expdev.nsls2.bnl.gov/static/js/ext/angular-1.6.1.min.js?20170214.000000:61:253",
        "ba@https://expdev.nsls2.bnl.gov/static/js/ext/angular-1.6.1.min.js?20170214.000000:59:81",
        "link@https://expdev.nsls2.bnl.gov/static/js/ext/angular-route-1.6.1.min.js?20170214.000000:7:266",
        "Anonymous function@https://expdev.nsls2.bnl.gov/static/js/ext/angular-1.6.1.min.js?20170214.000000:16:45",
        "ta@https://expdev.nsls2.bnl.gov/static/js/ext/angular-1.6.1.min.js?20170214.000000:84:35",
        "n@https://expdev.nsls2.bnl.gov/static/js/ext/angular-1.6.1.min.js?20170214.000000:69:226"
    ],
    "type": "clientException",
    "url": "https://expdev.nsls2.bnl.gov/srw#/beamline/L1lwwcrO"
}
```
That error message is accompanied by the empty "beamline definition area":
![screenshot](https://cloud.githubusercontent.com/assets/13209176/23825112/a6cecd18-0651-11e7-860d-45210ae890eb.png)

Unfortunately I cannot reproduce the error when I copy the simulation to a new browser.

**Comment by moellep** [comment]
I haven't been able to reproduce this error either. Does it happen each
time the beamline page is viewed? There may be a sequence of events which
triggers it.

**Comment by moellep** [comment]
I've tracked this down. This bug occurred because I didn't increase the version in schema-common.json when the beamline changes were checked in. The browser would have kept the sirepo-components.js cached and also loaded the new sirepo-beamline.js file and both would have defined the beamlineItemEditor directive. I've incremented the schema version with the latest check-in, so this won't be a problem with the next release.
